File: pescaria.cpp
#include <iostream>
using namespace std;

int main(){
    int e1, e2, res;
    cin >> e1;
    cin >> e2;
    res = e1*e2;

    if(500 > res){
        cout << "Paciencia Firmino!";
    } 
    else if(500 <= res && res < 1800){
        cout << "Vara de Bambu";
    }   
    else if(1800 <= res && res < 7500){
        cout << "Vara de Fibra de Vidro";
    }
    else{
        cout << "Vara de Iridio";
    }
}

File: pescaria/pescaria.py
# Vara de Bambu 500
# Vara de Vidro 1800
# Vara de Iridio 7500
# Paciencia Firmino!

entrada1 = int(input())
entrada2 = int(input())
res = entrada1 * entrada2

if 500 > res:
    print("Paciencia Firmino!")
elif 500 <= res < 1800:
    print("Vara de Bambu")
elif 1800 <= res < 7500:
    print("Vara de Fibra de Vidro")
else:
    print("Vara de Iridio")